src/platform/plugins/shared/data/server/search/session/session_service.ts

```diff
@@ -284,15 +283,12 @@ export class SearchSessionService implements ISearchSessionService {
   public trackId = async (
     deps: SearchSessionDependencies,
     user: AuthenticatedUser | null,
-    searchRequest: IKibanaSearchRequest,
     searchId: string,
     options: ISearchOptions
   ) => {
-    const { sessionId, strategy = ENHANCED_ES_SEARCH_STRATEGY } = options;
+    const { sessionId, strategy = ENHANCED_ES_SEARCH_STRATEGY, requestHash } = options;
     if (!this.sessionConfig.enabled || !sessionId || !searchId) return;
-    if (!searchRequest.params) return;
-
-    const requestHash = createRequestHash(searchRequest.params);
+    if (!requestHash) return;
```

Comment:
This seems like an error case. Maybe we can at least `logger.error` here with a little bit of info (like the `strategy` and `searchId`).